schema-engine/json-rpc-api/src/js_result.rs

```diff
@@ -8,25 +8,26 @@ use tsify_next::Tsify;
 #[derive(Debug, Copy, Clone, Serialize, Deserialize)]
 #[cfg_attr(target_arch = "wasm32", derive(Tsify))]
 #[serde(tag = "tag", content = "value", rename_all = "camelCase")]
-pub enum JSResult<R, E> {
+pub enum JsResult<R, E> {
```

Comment:
Let's use the `Js*` convention, as does `wasm_bindgen`, and as do other parts of our codebase (e.g., `JsQueryable`).